recipes/gcc/all/conanfile.py

```diff
@@ -155,11 +158,183 @@ def package(self):
+        if self.settings.os in ("Linux", "FreeBSD"):
+            self.cpp_info.components["asan"].system_libs.append("pthread")
+            self.cpp_info.components["asan"].system_libs.append("m")
+            self.cpp_info.components["asan"].system_libs.append("dl")
+
+        self.cpp_info.components["ubsan"].set_property("cmake_target_name", "gcc::ubsan")
+        self.cpp_info.components["ubsan"].libdirs = ["lib"]
+        self.cpp_info.components["ubsan"].libs = ["ubsan"]
+        self.cpp_info.components["ubsan"].requires = ["gcc_s", "stdc++"]
+        if self.settings.os in ("Linux", "FreeBSD"):
+            self.cpp_info.components["ubsan"].system_libs.append("pthread")
+            self.cpp_info.components["ubsan"].system_libs.append("dl")
+            self.cpp_info.components["ubsan"].system_libs.append("rt")
+
+        self.cpp_info.components["lsan"].set_property("cmake_target_name", "gcc::lsan")
+        self.cpp_info.components["lsan"].libdirs = ["lib"]
+        self.cpp_info.components["lsan"].libs = ["lsan"]
+        self.cpp_info.components["lsan"].requires = ["gcc_s", "stdc++"]
+        if self.settings.os in ("Linux", "FreeBSD"):
+            self.cpp_info.components["lsan"].system_libs.append("pthread")
+            self.cpp_info.components["lsan"].system_libs.append("dl")
+            self.cpp_info.components["lsan"].system_libs.append("rt")
+
+        self.cpp_info.components["cc1"].set_property("cmake_target_name", "gcc::cc1")
+        self.cpp_info.components["cc1"].libdirs = ["lib", "lib64"]
+        self.cpp_info.components["cc1"].libs = ["cc1"]
+        self.cpp_info.components["cc1"].requires = ["gcc_s", "stdc++"]
+
+        self.cpp_info.components["cp1plugin"].set_property("cmake_target_name", "gcc::cp1plugin")
+        self.cpp_info.components["cp1plugin"].libdirs = [os.path.join("lib","gcc", triplet, self.version, "plugin")]
+        self.cpp_info.components["cp1plugin"].libs = ["cp1plugin"]
+        self.cpp_info.components["cp1plugin"].requires = ["gcc_s", "stdc++"]
+
+        self.cpp_info.components["cc1plugin"].set_property("cmake_target_name", "gcc::cc1plugin")
+        self.cpp_info.components["cc1plugin"].libdirs = [os.path.join("lib","gcc", triplet, self.version, "plugin")]
+        self.cpp_info.components["cc1plugin"].libs = ["cc1plugin"]
+        self.cpp_info.components["cc1plugin"].requires = ["gcc_s", "stdc++"]
+
+        self.cpp_info.components["lto_plugin"].set_property("cmake_target_name", "gcc::lto_plugin")
+        self.cpp_info.components["lto_plugin"].libdirs = [os.path.join("bin","libexec", "gcc", triplet, self.version)]
+        self.cpp_info.components["lto_plugin"].libs = ["lto_plugin"]
+
+        self.cpp_info.components["gcov"].set_property("cmake_target_name", "gcc::gcov")
+        self.cpp_info.components["gcov"].libdirs = [os.path.join("lib","gcc", triplet, self.version)]
+        self.cpp_info.components["gcov"].libs = ["gcov"]
```

Comment:
It would be super cool to call this one in the the twst package!

I can see a bunch of people doing coverage with it

Reply:
I'll see if I can put something together